code/game/objects/items/devices/scanners.dm

```diff
@@ -202,7 +202,7 @@ REAGENT SCANNER
 //		user.show_message("\blue Bloodstream Analysis located [M.reagents:get_reagent_amount("epinephrine")] units of rejuvenation chemicals.")
 	if (M.has_brain_worms())
 		user.show_message("\red Subject suffering from aberrant brain activity. Recommend further scanning.")
-	else if (M.getBrainLoss() >= 100 || istype(M, /mob/living/carbon/human) && M:brain_op_stage == 4.0)
+	else if (M.getBrainLoss() >= 100 || istype(M, /mob/living/carbon/human) && !M.get_int_organ(/obj/item/organ/internal/brain))

```

Comment:
Only one species with that, and that is the IPC brain, which should be kinda maybe be a subtype, because then you can implant posibrains into people and they will live!